srtcore/api.cpp

```diff
@@ -2341,7 +2341,7 @@ CUDT* CUDT::getUDTHandle(UDTSOCKET u)
    {
       return s_UDTUnited.lookup(u);
    }
-   catch (std::exception& ee)
+   catch (...)
```

Comment:
Should this be "catch (CUDTException e) {} catch (std::exception& ee) {}" instead to be more consistent with the rest of the code or is there a reason to catch everything in this particular case?
Also I think we should apply the same change to getsockstate() just in case CUDTUnited::getStatus() implementation changes to throw CUDTException().